Tests/DependencyInjection/MremiContactExtensionTest.php

```diff
@@ -141,58 +141,59 @@ public function testContactLoadThrowsExceptionIfMailerEmpty()
     }
 
     /**
-     * Tests extension loading throws exception if from address is not set
+     * Tests extension loading throws exception if to address is not set
      *
      * @expectedException        \Symfony\Component\Config\Definition\Exception\InvalidConfigurationException
-     * @expectedExceptionMessage The child node "address" at path "mremi_contact.email.from.0" must be configured.
+     * @expectedExceptionMessage The child node "address" at path "mremi_contact.email.to.0" must be configured.
      */
-    public function testContactLoadThrowsExceptionUnlessFromAddressSet()
+    public function testContactLoadThrowsExceptionUnlessToAddressSet()
     {
         $loader = new MremiContactExtension;
         $config = $this->getFullConfig();
-        unset($config['email']['from'][0]['address']);
+        unset($config['email']['to'][0]['address']);
         $loader->load(array($config), new ContainerBuilder);
     }
 
     /**
-     * Tests extension loading throws exception if from address is empty
+     * Tests extension loading throws exception if to address is empty
      *
      * @expectedException        \Symfony\Component\Config\Definition\Exception\InvalidConfigurationException
-     * @expectedExceptionMessage The path "mremi_contact.email.from.0.address" cannot contain an empty value, but got "".
+     * @expectedExceptionMessage The path "mremi_contact.email.to.0.address" cannot contain an empty value, but got "".
      */
-    public function testContactLoadThrowsExceptionIfFromAddressEmpty()
+    public function testContactLoadThrowsExceptionIfToAddressEmpty()
     {
         $loader = new MremiContactExtension;
         $config = $this->getFullConfig();
-        $config['email']['from'][0]['address'] = '';
+        $config['email']['to'][0]['address'] = '';
         $loader->load(array($config), new ContainerBuilder);
     }
 
+
```

Comment:
Just create a new method like others and `unset($config['email']['to']);` and expect the good exception :)